tests/order_modifiers_integration/Coupons/Create_Coupon_Modifiers_Test.php

```diff
@@ -77,10 +76,17 @@ public function does_table_render_correctly_with_uses() {
 		$modifier_admin_handler = tribe( Modifier_Admin_Handler::class );
 		$_REQUEST['modifier']   = $this->modifier_type;
 
+		// Filter the per_page to 20.
+		$filter = static fn() => 20;
+		add_filter( 'coupon_per_page', $filter );
+
 		ob_start();
 		$this->get_table_class_instance()->prepare_items();
 		$modifier_admin_handler->render_tec_order_modifiers_page();
 
+		// Remove the filter
+		remove_filter( 'coupon_per_page', $filter );
+
```

Comment:
Filters should be reset on their own before every test case. That doesn't hurt anything just an info :D